Makefile

```diff
@@ -317,6 +319,19 @@ go-mod-tidy: ## Recursively run go mod tidy on all subdirectories
 check-mod-tidy: ## Recursively run go mod tidy on all subdirectories and check if there are any changes
 	@./control-plane/build-support/scripts/mod_tidy.sh --check
 
+.PHONY: go-mod-get
+go-mod-get: $(foreach mod,$(GO_MODULES),go-mod-get/$(mod)) ## Run go get and go mod tidy in every module for the given dependency
+
+.PHONY: go-mod-get/%
+go-mod-get/%:
+ifndef DEP_VERSION
+	$(error DEP_VERSION is undefined: set this to <dependency>@<version>, e.g. github.com/hashicorp/go-hclog@v1.5.0)
+endif
+	@echo "--> Running go get ${DEP_VERSION} ($*)"
+	@cd $* && go get $(DEP_VERSION)
+	@echo "--> Running go mod tidy ($*)"
+	@cd $* && go mod tidy
```

Comment:
I know this repo has `build-support/scripts/mod_tidy.sh`, but since we're running these two commands back-to-back and don't need `--check`, it seemed simpler and about as clean to do it inline here vs. calling that script as `go-mod-tidy` does.

Reply:
I agree. A lot of those scripts are pretty terrible too and should be replaces/deprecated...